src/nr_openai_observability/build_events.py

```diff
@@ -49,24 +68,12 @@ def build_events(response, request, response_headers):
         "api_type": response.api_type,
         "vendor": "openAI",
```

Comment:
Should be `OpenAI` what do we have in the Node SDK?

Reply:
This is the value in the node SDK as I commented below